private/bufpkg/bufcheck/bufbreaking/bufbreaking_test.go

```diff
@@ -114,9 +114,10 @@ func TestRunBreakingExtensionMessageNoDelete(t *testing.T) {
 		bufanalysistesting.NewFileAnnotation(t, "1.proto", 30, 3, 36, 4, "EXTENSION_MESSAGE_NO_DELETE"),
 		bufanalysistesting.NewFileAnnotation(t, "1.proto", 30, 3, 36, 4, "EXTENSION_MESSAGE_NO_DELETE"),
 		bufanalysistesting.NewFileAnnotation(t, "1.proto", 30, 3, 36, 4, "EXTENSION_MESSAGE_NO_DELETE"),
-		bufanalysistesting.NewFileAnnotation(t, "2.proto", 73, 1, 79, 2, "EXTENSION_MESSAGE_NO_DELETE"),
-		bufanalysistesting.NewFileAnnotation(t, "2.proto", 73, 1, 79, 2, "EXTENSION_MESSAGE_NO_DELETE"),
-		bufanalysistesting.NewFileAnnotation(t, "2.proto", 73, 1, 79, 2, "EXTENSION_MESSAGE_NO_DELETE"),
+		bufanalysistesting.NewFileAnnotation(t, "1.proto", 49, 1, 53, 2, "EXTENSION_MESSAGE_NO_DELETE"),
```

Comment:
Each of these adds a new problem, for the newly added `Ten` message. The other changes here are just shifting line numbers, since I added two lines towards the top in the `One` message.

The actual messages look like so:
```
Previously present extension range "[4,5]" on message "Seven" is missing values: [4,5] were removed
Previously present extension range "[1,max]" on message "Ten" is missing values: [1001,1999], [5001,9999], [50001,99999] were removed

```